countdown.py

```diff
@@ -0,0 +1,21 @@
+from command import Command
+from fbchat import Message
+from fbchat import Mention
+
+class countdown(Command):
+
+    def run(self):
+        count = int(self.user_params[0])
+        if count > 20:
+            response_text = "I'm too lazy to do that..." 
+        elif count <= 0:
+            response_text = "Lets do it again!"
+        else:
+            response_text = "!countdown" + str(count - 1)
```

Comment:
Add a space between the `n` and the `"` or else it won't be accepted as a valid command.